pkg/controller/plan/migration.go

```diff
@@ -336,6 +337,33 @@ func (r *Migration) Archive() {
 		return
 	}
 
+	if r.Plan.Provider.Source.Type() == v1beta1.Ova {
+		//delete pvc and pv
+		pvc, _, errPvc := GetOvaPvcNfs(r.Destination.Client, r.Plan.Name, r.Plan.Spec.TargetNamespace, r.Plan.Provider.Source.Name)
+		if errPvc != nil || pvc == nil {
+			r.Log.Error(err, "Failed to get the plan PVC")
```

Comment:
if there is no error and `pvc == nil` - is it an error that we didn't get it? it hints that the PVC doesn't exists (deleted already?)

Reply:
yeah it's mostly for the second case, that the PVC was already deleted and the error is not Found, in this case the GetOvaPvcNfs() isnt returning an error but we don't need to delete anything, I can change the check to be `!found` instead.

Reply:
that would be more logical to me
you can do also (i think it's ugly):
```
if errPvc != nil {
  r.log.Error(....
}
if pvc == nil {
  return
}
```
because we return `nil` if there is an error. but if we look on `found` i think it's nicer.